joomla/joomla-cms
# [4.0] Fix installer plugin detection

Redo PR #21262.

### Summary of Changes
Fix notice when no installer plugins.


### Testing Instructions
Under Manage > Plugins, search `Installer - Install`.
Disable 4 plugins.
Go to System > Install > Extensions.
See notice.

Install PR.
No more notice.


### Actual result
> Notice: Undefined offset: 0 in \administrator\components\com_installer\tmpl\install\default.php on line 43
